apps/wasm/task.py

```diff
@@ -269,7 +277,21 @@ def computation_finished(
         WasmTask.CALLBACKS[subtask_id] = verification_finished
 
         if subtask.is_finished():
-            self.__resolve_payments(subtask)
+            self._resolve_payments(subtask)
+
+            subtask_usages: List[UsageReport] = []
+            for s_id in subtask.get_instances():
+                subtask_instance = subtask.get_instance(s_id)
+                subtask_usages.append(
+                    (subtask_instance['actor'].uuid,
+                     s_id,
+                     task_result.stats.cpu_stats.cpu_usage.total_usage)
```

Comment:
* `cpu_usage` seems to be a dictionary
* `total_usage` seems to be in ns

```suggestion
                     task_result.stats.cpu_stats.cpu_usage['total_usage'] / 1e9 )
```